File: cache/tik.js


var tn=['a','b','c','d','e','f','g','h','i','j','k','l','m']
var Tn='ABCDEFGHIJKLM'

var rnd=(min,max)=>Math.floor(Math.random() * (max - min + 1) + min);

var random_key=()=>{
    var n=rnd(5,100)
    if (n % 2 == 1)
        n=n+1;
    var ts=[]
    for (let i=0;i<n/2;i++){
        ts[ts.length]=tn[rnd(0,tn.length-1)]
    }
    var Ts=''
    for (var i=0;i<n/2;i++){
        var p= rnd(1,Tn.length-1)
        Ts=Ts+Tn.charAt(p)
    }
    return ts.join('') +Ts
}

print=()=>{}

for (var i=1;i<=10000;i++){
    print(i,random_key())
}

File: cache/tik.lua

math.randomseed(os.time())
local tn={'a','b','c','d','e','f','g','h','i','j','k','l','m'}
local Tn='ABCDEFGHIJKLM'
local function random_key()
    local n=math.random(5,100)
    if n % 2 == 1 then
        n=n+1
    end
    local ts={}
    for i=1,n/2 do
        table.insert(ts,tn[math.random(1,#tn)])
    end
    local Ts=''
    for i=1,n/2 do
        local p=math.random(1,#tn)
        Ts=Ts..Tn:sub(p,p)
    end
    return table.concat(ts)..Ts
end

function print(...)
    
end

for i=1,10000 do
    print(i,random_key())
end